astroid/brain/brain_pathlib.py

```diff
@@ -27,10 +28,11 @@ def _looks_like_parents_subscript(node: nodes.Subscript) -> bool:
         value = next(node.value.infer())
     except (InferenceError, StopIteration):
         return False
+    parents = "builtins.tuple" if PY313_PLUS else "pathlib._PathParents"
```

Comment:
I'm wondering if this is too broad. Did we run `pylint` CI with this?

Reply:
Need to look closer, but at a glance nothing seems related: https://github.com/pylint-dev/pylint/actions/runs/9840792687/job/27165860010?pr=9776